tests/scripts/thread-cert/node.py

```diff
@@ -2159,6 +2159,21 @@ def reset(self):
         if self.is_otbr:
             self.set_log_level(5)
 
+    def factory_reset(self):
```

Comment:
The code is almost the same as `reset` above.
Maybe we can call `reset` with argument:
e.g. 

```suggestion
    def factory_reset(self):
        self._reset(cmd='factoryreset')
```